server/backend/users/tests/test_serializers_user.py

```
@@ -0,0 +1,43 @@
+        assert serializer.data['name'] == user.username
+        assert serializer.data['designation'] == instructor.designation
+        assert serializer.data['university'] == instructor.university
+
+    def test_instructor_serializer_missing_user(self):
+        instructor = Instructor(designation='Professor', university='University of Science')
+        try:
+            InstructorSerializer(instructor).data
+        except ValidationError as e:
+            assert 'user' in str(e)
+
+    def test_user_serializer_invalid_data(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_user_serializer_invalid_data`

Add return type annotation: `None`

(ANN201)